nats/rtt_command.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/nats-io/nats.go"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func configureRTTCommand(app *kingpin.Application) {
+	app.Command("rtt", "Compute round-trip time to NATS server").Action(rtt)
+}
+
+func rtt(pc *kingpin.ParseContext) error {
+	if !strings.Contains(servers, "://") {
+		servers = fmt.Sprintf("nats://%s", servers)
```

Comment:
server can be a list of servers it's passed directly to `nats.Connect`, so if we really want to go about prepending schema we need to do what processUrlString in the nats client does here too

Reply:
True.